lib/consumer.ex

```diff
@@ -27,7 +27,7 @@ defmodule GenRMQ.Consumer do
   ## Return values
   ### Mandatory:
 
-  `uri` - RabbitMQ uri
+  `connection` - RabbitMQ connection options. Accepts same options as AMQP-library's Connection.open().
```

Comment:
Let's add a link to the docs for `AMQP.Connection.open`, I know I would want it to know what options there are.
Same thing for the publisher docs (line 42).

Reply:
Added link to both consumer and publisher's docs.